Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/Constraints/ConstraintManager.cs

```diff
@@ -21,33 +21,48 @@ public ConstraintManager(GameObject gameObject)
             constraints = gameObject.GetComponents<TransformConstraint>().ToList();
         }
 
-        public void ApplyScaleConstraints(ref MixedRealityTransform transform)
+        public void ApplyScaleConstraints(ref MixedRealityTransform transform, bool isOneHanded, bool isNear)
```

Comment:
What are your thoughts on taking a ManipulationHandFlags directly? i.e. from a code readability perspective, it's easier to read a function like this:

ApplyScaleConstraints(ManipulationHandFlags.OneHanded) 

Over

ApplyScaleConstraints(true)

I think that there is a caveat here that it IS a flag (so passing both actually doesn't make sense)  

isNear is a bit weirder because the the places where it's getting called actually are already readable because they use a helper (i.e. isNearInteraction)

Anyways, not a huge thing, just curious to hear your thoughts.

Reply:
Yeah, I agree from a readability perspective, using the flags would be nicer. Also, we immediately convert the bools back  to `ManipulationHandFlags` and `ManipulationProximityFlags`, which we wouldn't have to do if they were parameters already. 

But the fact that passing `OneHanded | TwoHanded` or `Near | Far` doesn't make sense here is why I chose bools. I'm just not certain what the function should do in these cases where invalid flags are passed. Open to suggestions.